### PR TITLE
Add isAlias and getAlias functions

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Implement #20.

## Changes

Added `isAlias` and `getAlias` functions, and tests for them.
~~`getAlias` is made so it can treat the canonical address as an alias, by starting the counter from 0,
and otherwise start it from 1.~~

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

**Review by abizjak — APPROVED** [review]

File: src/alias.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+export function getAlias(
+    address: AccountAddress,
+    counter: number
+): AccountAddress {
+    if (counter < 0 || counter > maxCount) {
+        throw new Error('counter exceed maximum number of aliases');
+    }
+    const commonBytes = address.decodedAddress.slice(0, commonBytesLength);
+    const aliasBytes = Buffer.alloc(aliasBytesLength);
```
Comment (orhoj) [inline src/alias.ts:46]: Please use `buffer/` instead of the node Buffer.

File: test/alias.test.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    const address = new AccountAddress(
+        '4hXCdgNTxgM7LNm8nFJEfjDhEcyjjqQnPSRyBS9QgmHKQVxKRf'
+    );
+    const alias = getAlias(address, 1);
+    const otherAlias = getAlias(address, 100);
+    expect(otherAlias.address).not.toBe(alias.address);
+    expect(isAlias(otherAlias, alias)).toBeTruthy();
+});
+
+test('getAlias: last 3 bytes of alias matches counter', () => {
```
Comment (orhoj) [inline test/alias.test.ts:96]: I think this test with the minimum and maximum counter values would be good to have too.

**Review by orhoj — CHANGES_REQUESTED** [review]

File: test/alias.test.ts (hunk 2 above)
Comment (shjortConcordium) [reply test/alias.test.ts:96]: Added them to that test

**Comment by orhoj** [comment]
Please update the CHANGELOG with an entry for the added functionality.

**Review by orhoj — CHANGES_REQUESTED** [review]
Please update the CHANGELOG with the changes.

**Review by orhoj — APPROVED** [review]

**Pull request merged**